horizon/backend/db/repositories/user.py

```diff
@@ -18,13 +18,15 @@ async def count(self) -> int:
     async def get_by_id(self, user_id: int) -> User:
         result = await self._get_by_id(user_id)
         if result is None:
-            raise EntityNotFoundError("User", "id", user_id)
+            msg = "User"
+            raise EntityNotFoundError(msg, "id", user_id)
```

Comment:
That looks odd

Reply:
There is option `--errmsg-max-string-length` I can set it to 10 or mb more?

Reply:
set to 20